ql/src/java/org/apache/hadoop/hive/ql/ddl/view/create/CreateMaterializedViewDesc.java

```diff
@@ -237,11 +237,17 @@ public void setTblProps(Map<String, String> tblProps) {
     this.tblProps = tblProps;
   }
 
-  @Explain(displayName = "table properties")
   public Map<String, String> getTblProps() {
     return tblProps;
   }
 
+  @Explain(displayName = "table properties")
+  public Map<String, String> getTblPropsExplain() { // only for displaying plan
+    return PlanUtils.getPropertiesForExplain(tblProps,
+            hive_metastoreConstants.TABLE_IS_CTAS,
```

Comment:
should we rename the config to something like isCTASorCM or add a new one?

Reply:
Renamed the constant but kept the value since it would break compatibility.